src/common_modules/vsts-cli-release-common/vsts/cli/release/common/release_definition.py

```diff
@@ -0,0 +1,106 @@
+    :type artifact_source_id: str
+    :param detect: Automatically detect values for instance and project. Default is "on".
+    :type detect: str
+    :rtype: [ReleaseDefinitionReference]
+    """
+    team_instance, project = resolve_instance_and_project(detect=detect,
+                                                          team_instance=team_instance,
+                                                          project=project)
+    client = get_release_client(team_instance)
+    query_order = 'nameAscending'
+    definition_references = client.get_release_definitions(project=project, search_text=name, artifact_source_id=artifact_source_id,
+                                                    top=top,
+                                                    query_order=query_order)
+    return definition_references
+
+
+def release_definition_show(definition_id=None, name=None, open_browser=False, team_instance=None, project=None,
+                          detect=None):
+    """Get the details of a release definition.
+    :param definition_id: ID of the definition.
+    :type definition_id: int
+    :param name: Name of the definition. Ignored if --id is supplied.
+    :type name: str
+    :param open_browser: Open the definition summary page in your web browser.
+    :type open_browser: bool
+    :param team_instance: VSTS account or TFS collection URL. Example: https://myaccount.visualstudio.com
+    :type team_instance: str
+    :param project: Name or ID of the team project.
+    :type project: str
+    :param detect: Automatically detect values for instance and project. Default is "on".
+    :type detect: str
+    :rtype: ReleaseDefinitionReference
+    """
+    team_instance, project = resolve_instance_and_project(detect=detect,
+                                                            team_instance=team_instance,
+                                                            project=project)
+    client = get_release_client(team_instance)
+    if definition_id is None:
+        if name is not None:
+            definition_id = get_definition_id_from_name(name, client, project)
+        else:
+            raise ValueError("Either the --id argument or the --name argument must be supplied for this command.")
+    release_definition = client.get_release_definition(definition_id=definition_id, project=project)
+    if open_browser:
+        _open_definition(release_definition)
+    return release_definition
+
+
+def _open_definition(definition):
+    """Opens the release definition in the default browser.
+    :param :class:`<ReleaseDefinitionReference> <release.v4_0.models.ReleaseDefinitionReference>` definition:
+    """
+    url = _get_release_definition_web_url(definition)
+    if url is not None and url:
+        logger.debug('Opening web page: %s', url)
+        open_new(url=url)
+
+
+def get_definition_id_from_name(name, client, project):
```

Comment:
nit: naming - Function for use in this file only. Use _get_definition_id_from_name. (_function_name)

Reply:
This function is used in release.py file